database/postgres/postgres.go

```diff
@@ -1,3 +1,4 @@
+// +build go1.9
```

Comment:
We need to [support Go 1.8](https://golang.org/doc/devel/release.html#policy) until Go 1.10 is released in [Feb 2018](https://tip.golang.org/doc/go1.10#introduction)

So we can either:
1. Wait until Go 1.10 is release and we can drop Go 1.8 support and remove this build tag
1. Or have a separate implementation of postgres.go which supports Go 1.8. See [net/context](https://github.com/golang/net/tree/master/context) for an example of a package supporting multiple Go versions